project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.10.7
+sbt.version=2.0.0-M4
```

Comment:
this currently breaks metals integration (even with BSP handled by sbt itself rather than metals) pending the follow-up of https://github.com/scalameta/metals/pull/7045 for metals